st2common/st2common/services/sensor_watcher.py

```diff
@@ -103,9 +103,8 @@ def stop(self):
 
     @staticmethod
     def _get_queue(queue_suffix):
-        if not queue_suffix:
-            # pick last 10 digits of uuid. Arbitrary but unique enough for the TriggerWatcher.
-            u_hex = uuid.uuid4().hex
-            queue_suffix = uuid.uuid4().hex[len(u_hex) - 10:]
-        queue_name = 'st2.sensor.watch.%s' % queue_suffix
+        queue_name = queue_utils.get_queue_name(queue_name_base='st2.sensor.watch',
```

Comment:
Probably a good idea to add a docstring to this function which clarifies that we always add a random suffix to the queue name and why.